app/models/todo.rb

```diff
@@ -16,6 +16,7 @@ class Todo < ApplicationRecord
 
   scope :completed, -> { where completed: true }
   scope :active, -> { where completed: false }
+  scope :older_than_two_weeks, -> { where "created_at <= ?", 2.weeks.ago.to_datetime }
```

Comment:
What would you think about updating to use Arel?

```ruby
scope :old, -> { where arel_table[:created_at].lteq(2.weeks.ago) }
```

Reply:
I've developed the habit of avoiding SQL strings in ActiveRecord queries whenever possible.